wbooks/src/utils/stringUtils.test.js

```diff
@@ -0,0 +1,14 @@
+import { getInitials, keepNumbers } from './stringUtils';
+
+describe('stringUtils', () => {
+  test('getInitials', () => {
+    const name = 'Santiago';
+    const res = getInitials(name);
+    expect(res).toBe(name[0]);
+  });
+  test('keepNumbers', () => {
+    const value = 24234;
+    const res = keepNumbers(value);
+    expect(typeof res).toBe('string');
```

Comment:
You shouldn't test the type of response, you should check the response itself.
```suggestion
    expect(res).toBe('24234');
```